### PR TITLE
WIP: added translations

### PR DESCRIPTION
**Pull request by melindamariapop**

### CODE REVIEW

File: app/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -1,74 +1,127 @@
...
+    security: "Security"
+    airport_transfer: "Free shuttle service"
+    text1: '<span><b>Parking<span class="main-green">OK</span></b></span> is open for business 24 hours/day, 7 days/week. So no matter the time of your departure/arrival flight we are here for you. Your parking spot and coffee are ready!'
+    text2: 'Your car is safe with us. <span><b>Parking<span class="main-green">OK</span></b></span> has surveillance cameras and security 24 hours/day, 7 days/week. Our fenced and well lighted parking lot is located next to the airport’s runway which raises the security level of our lot because the only way to get access in is through our check point.'
+    text3: "We offer free around-the-clock shuttle service. The shuttle to/from the airport takes about one minute. Upon your return just give us a call from the terminal and the shuttle will be on the way to pick you up."
+
+prices_section:
+    intro: "The lowest rates guaranteed! The parking lot is only 600 meters away from the CLuj Napoca International Airport! Free around the clock shuttle to/from the airport!"
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: Please correct to lowercase "L" in "CLuj"

File: app/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -1,74 +1,127 @@
...
-    text3: "Transferul intre parcare si terminalele de plecari-sosiri din aeroport il asiguram noi. Durata transferului este de aprox. 1 min. La intoarcere, trebuie doar sa ne sunati imediat ce ati ajuns in incinta aeroportului si masina care asigura transferul va fi trimisa dupa dumneavoastra in cel mai scurt timp posibil."
+    security: "Security"
+    airport_transfer: "Free shuttle service"
+    text1: '<span><b>Parking<span class="main-green">OK</span></b></span> is open for business 24 hours/day, 7 days/week. So no matter the time of your departure/arrival flight we are here for you. Your parking spot and coffee are ready!'
+    text2: 'Your car is safe with us. <span><b>Parking<span class="main-green">OK</span></b></span> has surveillance cameras and security 24 hours/day, 7 days/week. Our fenced and well lighted parking lot is located next to the airport’s runway which raises the security level of our lot because the only way to get access in is through our check point.'
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: I would refactor the last part of this translation to '...which raises the security level by having only one access point, through our check point.' 
This removes the duplicate "lot" and "our" usages.

File: app/Resources/translations/messages.en.yml [hunk 3]
```diff
@@ -1,74 +1,127 @@
...
+
+prices_section:
+    intro: "The lowest rates guaranteed! The parking lot is only 600 meters away from the CLuj Napoca International Airport! Free around the clock shuttle to/from the airport!"
+    content1:
+        intro: "Parking rates"
+        text1: "Our rates start at 25 RON/day and can go as low as 9 RON/day, depending on the number of days. We offer special discount for long-term parking of over 30 days. Please call for details."
+    content2:
+        intro: "Book a reservation online or by phone"
+        text1: 'You can book your reservation by filling up the form from the “Rezervation“ tab or by calling <span class="main-blue"><a class="mobilesOnly" href="tel: %phoneNumber%"> <b>%phoneNumber%</b></a></span>.'
+        text2: "<b>Rezervations need to be made at least 24 hours in advance.</b><br> For online rezervations you will receive an email confirmation with details regarding your reservation and directions to the site. Please allow 24 hours for your reservation to be confirmed.<br>"
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: For the last part, "directions to the site", it should be "to the parking lot", right?

File: app/Resources/translations/messages.en.yml [hunk 4]
```diff
@@ -1,74 +1,127 @@
...
+            subject: "Thank you for your reservation."
+            greeting: "Hi %name%,"
+            intro: "Your reservation is confirmed!"
+            message: "<b>Reservation details:</b><br>Name: %name%<br>Email: %email%<br>Phone No.: %phone%<br>Arrival Date/Hour: %arrivalDate%<br>Return Date/Hour: %departureDate%<br>License Plate: %carNumber%<br>Message: %message%"
+            last_text: "Should you need any further assistance please contact us at: 0770629071."
+            signature: 'Looking forward seeing you,<br>ParkingOK<br><br><small>Strada Traian Vuia nr 259<br>Cluj-Napoca<br>Phone No.: 0770629071<br>E-mail: parkingok.cluj@gmail.com<br>www.parkingok.ro<br><a href="https://www.google.com/maps/place/ParkingOK/@46.782845,23.695182,17z/data=!3m1!4b1!4m5!3m4!1s0x47490b5d86df381d:0x17a0e6400d7fbc28!8m2!3d46.782845!4d23.6973707?hl=it">Map</a></small>'
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: Shouldn't it be "Traian Vuia Street, number 259"? Thank you.

File: app/Resources/translations/messages.en.yml [hunk 5]
```diff
@@ -1,74 +1,127 @@
...
+    content2:
+        intro: "Book a reservation online or by phone"
+        text1: 'You can book your reservation by filling up the form from the “Rezervation“ tab or by calling <span class="main-blue"><a class="mobilesOnly" href="tel: %phoneNumber%"> <b>%phoneNumber%</b></a></span>.'
+        text2: "<b>Rezervations need to be made at least 24 hours in advance.</b><br> For online rezervations you will receive an email confirmation with details regarding your reservation and directions to the site. Please allow 24 hours for your reservation to be confirmed.<br>"
+    content3:
+        intro: "How are Daily Rates calculated"
+        text1: "The day starts at 12:00am and ends at 11:59pm. If you arrive after 9:00pm you will not be charged that day and/or if your pick up is between 12:00am and 3:00am you will not be charged for that day."
+        text2: "If your return date is earlier than the one listed on the contract you will be charged the number of days in contract. If your return date is later than the one listed on the contract your rates will be recalculated by the number of days used."
+        text3: ""
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: This does not have a value but it is used in twig, is this intended?

File: app/Resources/translations/messages.en.yml (hunk 1 above)
Comment (melindamariapop) [reply app/Resources/translations/messages.en.yml]: :D

**Pull request merged**